calico/felix/fetcd.py

```diff
@@ -352,7 +371,22 @@ def _load_config(self):
                            "retry.", e)
                 gevent.sleep(RETRY_DELAY)
             else:
-                self.config.report_etcd_config(host_dict, global_dict)
+                if self.configured.is_set():
+                    # We've already been configured.  We don't yet support
+                    # dynamic config update so instead we check if the config
+                    # has changed and die if it has.
+                    _log.info("Checking configuration for changes...")
+                    if (host_dict != self.last_host_config or
+                            global_dict != self.last_global_config):
+                        _log.critical("Felix configuration has changed, "
```

Comment:
I agree that this should be high priority, but not that high - I'd probably say error or warning.